dev-docs/show-prebid-ads-on-amp-pages.md

```diff
@@ -125,7 +125,38 @@ This is the creative that your Ad Ops team needs to upload to the ad server (it'
 {: .alert.alert-success :}
 You can always get the latest version of the creative code below from [the AMP example creative file in our GitHub repo](https://github.com/prebid/prebid-universal-creative/blob/master/template/amp/dfp-creative.html).
 
-{% include dev-docs/amp-creative.md %}
+{: .alert.alert-danger :}
+Replace **MACRO** in the code below with the appropriate macro for the ad server you're using. For example, for DFP replace **MACRO** with **PATTERN**.
+
+{% highlight html %}
+
+    <script src="https://cdn.jsdelivr.net/npm/prebid-universal-creative@latest/dist/creative.js"></script>
```

Comment:
No need to indent the entire code block.

Any reason why this code snippet is different from the snippet above?

Reply:
every code snippet on this page is indented like this, but I can un-indent this one if you prefer.

difference is html vs. javascript. I had this as html, I'll change to javascript.

Reply:
I think all of them should be aligned relative to the left edge of the code box.  Not just this one.